android/src/main/java/com/adyenreactnativesdk/component/AdyenInstantComponent.kt

```diff
@@ -0,0 +1,130 @@
+package com.adyenreactnativesdk.component
+
+import android.util.Log
+import com.adyen.checkout.components.ActionComponentData
+import com.adyen.checkout.components.model.payments.Amount
+import com.adyen.checkout.components.model.payments.request.GenericPaymentMethod
+import com.adyen.checkout.components.model.payments.request.PaymentComponentData
+import com.adyen.checkout.components.model.payments.request.PaymentMethodDetails
+import com.adyen.checkout.components.model.payments.response.Action
+import com.adyen.checkout.core.api.Environment
+import com.adyenreactnativesdk.*
+import com.adyenreactnativesdk.configuration.RootConfigurationParser
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.ReadableMap
+import org.json.JSONException
+import java.util.*
+
+class AdyenInstantComponent(context: ReactApplicationContext?) : BaseModule(context),
+    PaymentComponentListener, ActionHandlingInterface {
+
+    private var actionHandler: ActionHandler? = null
+
+    override fun getName(): String {
+        return "AdyenInstant"
```

Comment:
I think it would be better to have this string as a static, so have a `private const val` in the `companion object`

Reply:
👍